ui/app/registration/controllers/patientCommonController.js

```diff
@@ -190,6 +369,25 @@ angular.module('bahmni.registration')
             $scope.$watch('patientLoaded', function () {
                 if ($scope.patientLoaded) {
                     executeShowOrHideRules();
+                    if ($scope.patient.extraIdentifiers !== undefined && $scope.idStatus === false) {
+                        for (var i = 0; i < $scope.patient.extraIdentifiers.length; i++) {
```

Comment:
You can replace with array.forEach() .. also maybe better to extract to a function?